WordPress/Sniffs/Theme/NonPrintableCheckSniff.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+/**
+ * Restricts the use of non-printable characters in themes
```

Comment:
`in themes` can be removed. The sniff can be used on any file, not just WP theme files.